Mail/Views/New Message/AutocompletionView.swift

```diff
@@ -32,6 +33,8 @@ struct AutocompletionView: View {
     @Binding var autocompletion: [Recipient]
     @Binding var addedRecipients: RealmSwift.List<Recipient>
 
+    @LazyInjectService private var accountManager: AccountManager
```

Comment:
Not needed

Reply:
Yeah, they were needed, before I merged back master. I'm cleaning it 🧹